tensornetwork/network_components.py

```diff
@@ -1254,7 +1253,7 @@ def flatten_edges(edges: List[Edge],
     if axis_names:
       node.axis_names = [axis_names[n] for n in range(len(node.edges))]
     else:
-      node.axis_names = None
+      node.axis_names = ["__unnamed_edge__" for n in range(len(node.edges))]
```

Comment:
we should stick to the same convention as in BaseNode.__init__

Reply:
or at least make axis_names unique (looks like a typo though)

Reply:
Good point. Switched to `str(n)`